modules/ROOT/pages/intro-error-handlers.adoc

```diff
@@ -84,7 +84,9 @@ For the most part, Mule 3 only allows error handling at the flow level, forcing
 
 image::error-handling-try.png[Try Scope]
 
-The error handler behaves as we have explained earlier. In the example above, any database connection errors are propagated, causing the `try` to fail and the flow’s error handler to execute. In this case, any other errors are handled, and the Try scope is considered successful which, in turn, means that the next processor in the flow, an HTTP request, continues its execution.
+In the above example, an on error propagate handler is configured to propagate database connection errors, causing the `Try` to fail and the flow's error handler to execute. 
```

Comment:
On Error Propagate component. 

See the docs for the appropriate spelling and capitalization. 

`Try` -> Try scope

Reply:
Got it! Txs!!